test/run_readelf_tests.py

```diff
@@ -142,13 +145,27 @@ def prepare_lines(s):
         return False, 'Number of lines different: %s vs %s' % (
                 len(lines1), len(lines2))
 
+    view_col_position = None
     for i in range(len(lines1)):
         if lines1[i].endswith('debug_line section:'):
             # .debug_line or .zdebug_line
             flag_in_debug_line_section = True
 
         # readelf spelling error for GNU property notes
         lines1[i] = lines1[i].replace('procesor-specific type', 'processor-specific type')
+        
+        # The view column position may change from CU to CU:
+        if view_col_position and lines1[i].startswith('cu:'):
+            view_col_position = None    
+    
+        # Check if readelf..decodedline output contains the view column
+        if flag_in_debug_line_section and lines1[i].startswith('file name') and view_col_position is None:
+            view_col_position = lines1[i].find("view")
+            stmt_col_position = lines1[i].find("stmt")
+
+        # Excise the View column, if any
+        if view_col_position and not lines1[i].endswith(':'):
+            lines1[i] = lines1[i][:view_col_position] + lines1[i][stmt_col_position:]
```

Comment:
What will happen if stmt_col_position wasn't defined? (the `if` before this one didn't fire)

Reply:
Then the part of the line that corresponds to the View won't be cut from it, because that means there is no view column, of the file is not a decodedlines file to begin with :)

The if that sets view_col_position fires for the header line (, the next if fires for both header and every table line below, until the next CU, where view_col_position is reset back to None until the next table header.

Reply:
It would be clearer if `stmt_col_position` would be defined with some default value outside the condition. You can add an assert for good measure if needed. Python's "define stuff inside blocks to be visible outside them" is not very readable.

Reply:
Okay, I'll ditch the reliance on None.